modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyRequest < ApplicationRecord
+    belongs_to :claimant,
+               class_name: 'UserAccount'
+
+    has_one :form,
+            class_name: 'AccreditedRepresentativePortal::PowerOfAttorneyForm',
+            inverse_of: :power_of_attorney_request,
+            dependent: :destroy
+
+    has_one :resolution,
```

Comment:
removing redundant suffix is good here though, i.e. `power_of_attorney_request.resolution` is the right expression in my opinion.